bin/revision.py

```diff
@@ -165,14 +172,18 @@
                            "-I", """Thu, """, "-I", """Fri, """,
                            "-I", """Sat, """,
                            options.output + 'Try.hxxout',
-                           options.output + '.hxxout'], stdout=f_null)
+                           options.output + '.hxxout'],
+                          stdout=f_null, stderr=f_null)
 diffcxx = subprocess.call(['diff',
                            "-I", """Sun, """, "-I", """Mon, """,
                            "-I", """Tue, """, "-I", """Wed, """,
                            "-I", """Thu, """, "-I", """Fri, """,
                            "-I", """Sat, """,
                            options.output + 'Try.cxxout',
-                           options.output + '.cxxout'], stdout=f_null)
+                           options.output + '.cxxout'],
+                          stdout=f_null, stderr=f_null)
+# disable this because we are not actually writing a cxx file above.
+diffcxx = 0
```

Comment:
Does this result in dead code that should be removed from the script?

Reply:
I think you might be better in advising on this question as the author of this script. The script seems to be written for generating an hxx and a cxx file, but it only generates one with the concatenation of the code for both. Why is this the case? The comments in the code did not give a hint. So this can be cleaned up in two ways and I would hope you can decide which way to go:
1. Make it actually generate a cxx file.
2. Clean up the code to that was there to generate the cxx file.

Reply:
I'll merge this now as is and we can discuss which way to move forward.